uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/my-leave/app.js

```diff
@@ -8,7 +8,7 @@ define([
   'common/services/angular-date/date-format',
   'leave-absences/shared/ui-router',
   'leave-absences/my-leave/modules/config',
-  'leave-absences/my-leave/components/my-leave',
+  'leave-absences/my-leave/components/my-leave-container',
```

Comment:
The error happens when we have same file name for multiple components.
Added `-container` to all files(Html, Component, test etc.)